src/codelens/bazel_build_code_lens_provider.ts

```diff
@@ -161,22 +171,15 @@ export class BazelBuildCodeLensProvider implements vscode.CodeLensProvider {
         });
       }
 
-      // All targets support building.
-      commands.push({
-        commandString: "bazel.buildTarget",
-        name: "Build",
-      });
-
       for (const command of commands) {
-        const title = `${command.name} ${targetShortName}`;
```

Comment:
note that there might be multiple targets on the same line, e.g., if you use `cc_binary` in a list construction